spec/order_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'order'
+
+describe Order do
+  it "Has a method select dishes with 2 arguments"do
+    expect(subject).to respond_to(:select_dishes).with(3).arguments
+  end
+
+  it "Can select a number dishes off the menu" do
+    subject.select_dishes("hamburger",6.0, 2)
+    expect(subject.ticket).to eq [{"hamburger"=>6.0}, {"hamburger"=>6.0}]
```

Comment:
Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.